CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing to Folly
+We want to make contributing to this project as easy and transparent as
+possible.
+
+## Code of Conduct
+The code of conduct is described in [`CODE_OF_CONDUCT.md`](CODE_OF_CONDUCT.md).
+
+## Pull Requests
+We actively welcome your pull requests.
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
```

Comment:
Linted with what? Seems like `Folly` does not provide a `.clang-format` file or auto formatting or other linting, or am I missing something?

Reply:
Ah - good catch. Didn't realize, will fix. :)